### PR TITLE
Allows empty list of currency pairs

### PR DESCRIPTION
**Pull request by gip**

### CODE REVIEW

**Comment by gip** [comment]
cc @lumberj and @MatthewPhinney

File: src/backends/fixerio/index.js [hunk 1]
```diff
@@ -32,9 +31,6 @@ class FixerIoBackend {
     this.rates = {}
     this.currencies = []
     const ledgerPairs = config.get('tradingPairs')
-    if (_.isEmpty(ledgerPairs)) {
```
Comment (MatthewPhinney) [inline src/backends/fixerio/index.js]: This was a "safety check" that Chris and I decided to add. Live and learn, I guess ...

**Comment by MatthewPhinney** [comment]
LGTM

File: src/backends/fixerio/index.js (hunk 1 above)
Comment (clark800) [inline src/backends/fixerio/index.js]: I meant that we should check for `null`/`undefined`, sorry I should have caught this in the review

Comment (clark800) [inline src/backends/fixerio/index.js]: Still think we should check for `null`/`undefined` though

Comment (gip) [inline src/backends/fixerio/index.js]: Makes sense. I'll add that when I work on the unit tests.

**Pull request merged**